benchmarks/image-rotate/README.md

```diff
@@ -0,0 +1,64 @@
+# Image Rotate Benchmark
+
+The image rotate benchmark rotates an input image by 90 degrees. An input image can be specified, but if nothing is given, a default image is used. This benchmark also utilises and depends on a database to store the images that can be used, which in turn uses MongoDB. 
```

Comment:
You could add something like this: The `init-database.go` script runs when starting the function and populates the database with the images from the `images` folder.